app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count + 1)
+  end
```

Comment:
- [ ] OPTIONAL
Again, `update_like_counter` should be marked as _private_ after implementing the `after_save` method.